src/output-view.js

```diff
@@ -0,0 +1,109 @@
+import {
+    Transformime,
+    TextTransformer,
+    ImageTransformer,
+    HTMLTransformer
+} from "transformime";
+
+import {
+    consoleTextTransform,
+    markdownTransform,
+    LaTeXTransform,
+    PDFTransform
+} from "transformime-jupyter-transformers";
+
+/**
+ * Displays output area state.
+ */
+export class OutputView {
+    
+    /**
+     * Public constructor
+     * @param  {IOutputModel} model    output model that this view represents
+     * @param  {Document}     document handle to the Document instance that 
+     *                                 the output will be rendered in.
+     */
+    constructor(model, document) {
```

Comment:
Ooh, so glad you pass a document so we can try iframe experiments with this too!